src/services/EdFi.DataManagementService.Api/appsettings.json

```diff
@@ -1,7 +1,8 @@
 {
     "AppSettings": {
         "BeginAllowedSchoolYear": 2022,
-        "EndAllowedSchoolYear": 2035
+        "EndAllowedSchoolYear": 2035,
+        "AuthenticationService": ""
```

Comment:
Suggest putting in a dummy URL in the appsettings.Development.json file since we shouldn't allow `null`. The actual value doesn't really matter right now.